app/services/submit_application.rb

```diff
@@ -4,6 +4,13 @@ def initialize(url, token)
     @token = token
   end
 
+  def available?
```

Comment:
It's an interesting point. 

Before allowing them to start an application, I would say yes. 
Before submission it's probably not necessary as the actually submission would fail anyway. So the call to the api would be unnecessary duplicated (even though different type).

Let's bring it up with Adrian / Jason.